tap_gorgias/__init__.py

```diff
@@ -13,7 +13,7 @@
 
 REQUIRED_CONFIG_KEYS = ["subdomain", "username", "password", "start_date"]
 SUB_STREAMS = {
-    'tickets': ['messages']
+    # 'tickets': ['messages']
```

Comment:
Messages are longer dependant on tickets

Reply:
can we go ahead and delete this entire dict then?

Reply:
I will empty it instead, don't want to delete the logic for substreaming in case we need it in the future.